Sources/Build/BuildDescription/ProductBuildDescription.swift

```diff
@@ -193,6 +193,12 @@ public final class ProductBuildDescription: SPMBuildCore.ProductBuildDescription
         // radar://112671586 supress unnecessary warnings
         if triple.isMacOSX {
             args += ["-Xlinker", "-no_warn_duplicate_libraries"]
+        } else if triple.isWindows() {
+            // locally defined symbol imported
+            // Occurs when an object file exports a symbol is imported
+            // into another module in the same executable.
+            // FIXME: Need to support targets as DLLs
+            args += ["-Xlinker", "/ignore:4217"]
```

Comment:
Please do not do this - it is hiding a real issue.

Reply:
The ones I'm seeing aren't actionable by the user. They're caused by our lack of support for dynamic targets. There's so many you miss any other warning that might be actionable.

If a target appears in a DLL, it doesn't get the -static arg. But then if you include it in a EXE, like tests, you get the warning.

Reply:
@compnerd brings up a good point about hiding user caused warnings. I'll take this out for now and see what the fallout is. And we'll continue to work to make sure SwiftPM isn't causing these warnings.